src/accordion/accordion.ts

```diff
@@ -45,27 +17,48 @@ export class NgbAccordion {
     </div>
   `
 })
-export class NgbAccordionGroup implements OnDestroy {
-  private isDisabled: boolean;
-  private _isOpen: boolean = false;
+export class NgbAccordionGroup {
+  private _isOpen = false;
+  private accordion: NgbAccordion;
+  @Input() isDisabled: boolean;
+  @Input() heading: string;
+
+  @Input()
+  set isOpen(value: boolean) {
+    this._isOpen = value;
+    if (value) {
+      this.accordion.closeOthers(this);
```

Comment:
Soooo, this is a circular dependency (accordion -> group and group -> accordion). Ideally we should remove this circle, but I'm not sure how atm... Let me fiddle with different ideas....